python/lsst/ts/observatory/control/mock/mtcs_mock.py

```diff
@@ -600,126 +599,112 @@ async def m1m3_telemetry(self):
-                    accelerometer_data
+                await self.controllers.mtm1m3.tel_accelerometerData.set_write(
+                    timestamp=current_tai(),
+                    rawAccelerometer=raw_accelerometer_arr,
+                    accelerometer=raw_accelerometer_arr,
                 )
 
                 force_actuator_data = (
                     self.controllers.mtm1m3.tel_forceActuatorData.DataType()
                 )
 
-                force_actuator_data.timestamp = current_tai()
-
-                force_actuator_data.primaryCylinderForce = np.random.rand(
-                    len(force_actuator_data.primaryCylinderForce)
-                )
-                force_actuator_data.secondaryCylinderForce = np.random.rand(
-                    len(force_actuator_data.secondaryCylinderForce)
-                )
-                force_actuator_data.xForce = np.random.rand(
-                    len(force_actuator_data.xForce)
-                )
-                force_actuator_data.yForce = np.random.rand(
-                    len(force_actuator_data.yForce)
-                )
-                force_actuator_data.zForce = np.random.rand(
-                    len(force_actuator_data.zForce)
-                )
-                force_actuator_data.fx = np.random.rand()
-                force_actuator_data.fy = np.random.rand()
-                force_actuator_data.fz = np.random.rand()
-                force_actuator_data.mx = np.random.rand()
-                force_actuator_data.my = np.random.rand()
-                force_actuator_data.mz = np.random.rand()
-
-                force_actuator_data.forceMagnitude = np.sqrt(
-                    force_actuator_data.fx**2
-                    + force_actuator_data.fy**2
-                    + force_actuator_data.fz**2
-                )
+                fx = np.random.rand()
+                fy = np.random.rand()
+                fz = np.random.rand()
 
-                await self.controllers.mtm1m3.tel_forceActuatorData.write(
-                    force_actuator_data
+                await self.controllers.mtm1m3.tel_forceActuatorData.set_write(
+                    timestamp=current_tai(),
+                    primaryCylinderForce=np.random.rand(
+                        len(force_actuator_data.primaryCylinderForce)
+                    ),
+                    secondaryCylinderForce=np.random.rand(
+                        len(force_actuator_data.secondaryCylinderForce)
+                    ),
+                    xForce=np.random.rand(len(force_actuator_data.xForce)),
+                    yForce=np.random.rand(len(force_actuator_data.yForce)),
+                    zForce=np.random.rand(len(force_actuator_data.zForce)),
+                    fx=fx,
+                    fy=fy,
+                    fz=fz,
+                    mx=np.random.rand(),
+                    my=np.random.rand(),
+                    mz=np.random.rand(),
+                    forceMagnitude=math.hypot(fx, fy, fz),
```

Comment:
NumPy is already used throughout the code.. can you use `np.hypot` instead of `math`?